docs/user_manual/working_with_ogc/ogc_client_support.rst

```diff
@@ -945,6 +945,112 @@ you can install the QGIS plugin :guilabel:`Data Plotly`.
 
 
+STAC (SpatioTemporal Asset Catalogs)
+====================================
+
+.. index:: STAC
+.. _stac:
+
+STAC (SpatioTemporal Asset Catalog) is a specification for describing geospatial
+data in a consistent and accessible way. It defines a standard structure for organizing
+and indexing spatial-temporal assets such as satellite imagery, drone photos, or sensor data
+so that you can search, preview, and use these datasets across different platforms and tools.
+
+In QGIS, STAC support is integrated as a native data provider,
+allowing users to connect to and browse STAC catalogs directly within the interface.
+STAC catalogs contain structured metadata about each dataset, including spatial and
+temporal coverage, asset type, and related properties. This metadata can be used to
+search for relevant datasets based on criteria like time of acquisition or geographic extent.
+
+Key components of STAC in QGIS include:
+
+* **STAC Items**: These represent one or more geospatial assets tied to a specific location and time.
+  For example, a satellite image may include multiple bands, each as a separate asset within the same item.
+* **STAC Collections**: Groups of related items that share common metadata.
+* **STAC Catalogs**: These are hierarchical structures that organize collections and items,
+  allowing users to navigate through datasets easily.
+
+
+Setting connection
+------------------
+
+STAC connection can be added through either the :guilabel:`Browser panel` or the :guilabel:`Data Source Manager`:
+
+* **Browser Panel:**
+  In the :guilabel:`Browser`, right-click the |stac|:guilabel:`STAC` entry and select :guilabel:`New STAC Connection…`.
+  In the dialog that appears, enter a :guilabel:`Name` for the connection and the :guilabel:`URL` of the STAC
+  endpoint, then click :guilabel:`OK`.
+
+* **Data Source Manager:**
+  Open the |dataSourceManager|:guilabel:`Data Source Manager`, choose the |stac|:guilabel:`STAC` tab and click the :guilabel:`New Connection` button.
+  Fill in the :guilabel:`Name` and :guilabel:`URL` fields, press :guilabel:`OK` and then
+  :guilabel:`Connect` to establish the connection, after that you will be able to:
+
+   * :guilabel:`Edit` the STAC connection settings
+   * :guilabel:`Remove` the STAC connection
+
+  .. figure:: img/stac_connection.png
+     :align: center
+
+     Creating a connection to a STAC server
+
+Browsing STAC Catalogs
+----------------------
+
+Once connected, the STAC catalog appears under |stac|:guilabel:`STAC` in the :guilabel:`Browser` panel
+or in the |dataSourceManager|:guilabel:`Data Source Manager` interface.
+You can expand the catalog node to see its :guilabel:`Collections`.
+Expanding a collection reveals the individual :guilabel:`Items` it contains.
+
+ .. figure:: img/stac_browser.png
+    :align: center
+
+    STAC connection expanded in the Browser, showing Collections and Items
+
+Right-click any STAC :guilabel:`Item` and choose :guilabel:`Details...` to view its metadata.
```

Comment:
I get the same entries from the DSM dialog, I thought these were collections. so I'm a bit lost: what is an item, what is a collection, and so on...